crates/hyperswitch_interfaces/Cargo.toml

```diff
@@ -25,6 +26,9 @@ thiserror = "1.0.58"
 time = "0.3.35"
 
 # First party crates
+actix-web = "4.5.1"
```

Comment:
Please move this out of First party crates

Reply:
Done